### ISSUE TITLE
Column Width Issue in Addresses Tab

### ISSUE THREAD

**Issue by Ardakilic** [spesmilo/electrum#3847]
Hello,

I'm having an issue with the column widths in the Addresses tab.

I want to Widen the Balance and Txs columns, but they are locked and I cannot do it.

![image](https://user-images.githubusercontent.com/2063957/35812276-880dc8ac-0aa1-11e8-82b3-afa5740ea00a.png)

The last 2 columns' widths are locked. I want to widen them. There's a balance but because width is narrow I cannot see it.

I tried dragging, etc. No help.

I'm using Electrum 3.06 (latest version as of now) on Windows (portable) and macOS High Sierra, both have the same issue. Wallet was upgraded from 2.x versions.

I believe I'm missing something basic here, but I'm stuck.

I'd appreciate if you could assist me here.

Thanks i advance,
Kind Regards,

**Comment by SomberNight** [comment]
Are you sure there is a balance on that address?
Try changing the filter to `Funded`. Or look for your coins among `Change` addresses.
You can also right-click>`Details` an address btw.

**Comment by Ardakilic** [comment]
Ohh, 

Indeed you are right sir, They are not in `Funded` tab, but when I set the filter from  `Receiving` to `Change`, I can see my balance. I guess it's because I made a payment over BitPay recently (first time doing this). and my balance changed to a new address.

The column is It's still locked, but at least I can see my balance not trimmed out, which is good enough for me.

Thank you @SomberNight for directing to the right way, this issue can be marked as resolved.
